packages/itwinui-variables/src/themes/dark.scss

```diff
@@ -164,4 +166,6 @@ $opacities: (
   --iui-color-warning-hsl: #{get-hsl($fg-warning-2)};
   --iui-color-negative-hsl: #{get-hsl($fg-negative-2)};
   // #endregion
+
+  --iui-shadow-color: #{hsl(0 0% 0% / var(--iui-opacity-4))};
```

Comment:
To confirm, `--iui-shadow-color` is not needed to be defined in `dark-hc` and `light-hc`?

Reply:
`light-hc` and `dark-hc` are built on top of `light` and `dark` respectively, so not needed unless the values should be different